viewer/packages/utilities/src/shapes/Box.ts

```diff
@@ -23,7 +23,7 @@ export class Box implements IShape {
   }
 
   createBoundingBox(): Box3 {
-    const baseBox = new Box3(new Vector3(-0.5, -0.5, -0.5), new Vector3(0.5, 0.5, 0.5));
+    const baseBox = new Box3(new Vector3(-1.0, -1.0, -1.0), new Vector3(1.0, 1.0, 1.0));
```

Comment:
Also, we should mention somewhere in TSDoc or in actual doc what we mean by unit box, i.e. state exact dimensions for annotation boxes. Probably mentioning that we premultiply them with some value is also good.

Reply:
Jotted it down in the `data-providers` README